modules/olmv1-red-hat-catalogs.adoc

```diff
@@ -11,50 +11,70 @@
 
 {olmv1-first} includes the following Red Hat-provided Operator catalogs on the cluster by default. If you want to add an additional catalog to your cluster, create a custom resource (CR) for the catalog and apply it to the cluster. The following custom resource (CR) examples show the default catalogs installed on the cluster.
 
-.Example Red Hat Operators catalog
+.Red{nbsp}Hat Operators catalog
 [source,yaml,subs="attributes+"]
 ----
-apiVersion: catalogd.operatorframework.io/v1alpha1
+apiVersion: olm.operatorframework.io
 kind: ClusterCatalog
 metadata:
-  name: redhat-operators
+  name: openshift-redhat-operators
```

Comment:
I'm assuming you confirmed the names?

Reply:
I used these manifests for reference:
* https://github.com/openshift/operator-framework-catalogd/blob/main/openshift/manifests/15-clustercatalog-openshift-certified-operators.yml
* https://github.com/openshift/operator-framework-catalogd/blob/main/openshift/manifests/16-clustercatalog-openshift-community-operators.yml
* https://github.com/openshift/operator-framework-catalogd/blob/main/openshift/manifests/17-clustercatalog-openshift-redhat-marketplace.yml
* https://github.com/openshift/operator-framework-catalogd/blob/main/openshift/manifests/18-clustercatalog-openshift-redhat-operators.yml